electron/package.json

```diff
@@ -58,6 +61,12 @@
     "url": "https://github.com/wireapp/wire-desktop.git"
   },
   "scripts": {
+    "coverage:check": "istanbul check-coverage --branches=10 --functions=10 --lines=35 --statements=30 instrumented-code/coverage.json",
+    "coverage:clean": "rimraf coverage instrumented-code",
+    "coverage:instrument": "istanbul instrument dist --no-compact --complete-copy --output instrumented-code",
+    "coverage:report": "remap-istanbul --input instrumented-code/coverage.json --output coverage -t html",
+    "coverage:test": "electron-mocha",
```

Comment:
maybe this script can be renamed just `test`. It is not directly linked to coverage, am I wrong?

Reply:
Ok, has been renamed: https://github.com/wireapp/wire-desktop/pull/2215/commits/6f4f2e6bf583c6f320948b01603936ecfaad210b